packages/svelte/src/internal/server/crypto.js

```diff
@@ -0,0 +1,13 @@
+let text_encoder;
+// TODO - remove this and use global `crypto` when we drop Node 18
+let crypto;
+
+/** @param {string} data */
+export async function sha256(data) {
+	text_encoder ??= new TextEncoder();
+	// @ts-ignore - we don't install node types in the prod build
+	crypto ??= (await import('node:crypto')).webcrypto;
```

Comment:
come to think of it didn't we talk about skipping this and assuming that `crypto` is always available as a global, given that `hydratable` is behind the `experimental.async` flag? not worried about excluding people who want to use bleeding edge Svelte features but are still rocking Node 16 or whatever

Reply:
Can't we just do `globalThis.crypto ??= await import('node:crypto').webcrypto` in the tests, before this module is imported?